d3a_api_client/utils.py

```diff
@@ -77,7 +81,7 @@ def get_request(endpoint, data, jwt_token):
 
 
 def request_response_returns_http_200(endpoint, resp):
-    if resp.status_code != 200:
+    if abs(resp.status_code % 200) > 100:
```

Comment:
Care to elaborate here? This will accept 400-499 response codes, if I can read the code correctly.

Reply:
Darn, you are right. I wanted to allow http codes like 201, too.

Reply:
Changed it: https://github.com/gridsingularity/d3a-api-client/pull/54/commits/09c6609427960980afe219d1298693d5c08a3743